src/solver/impls/arkode/arkode.cxx

```diff
@@ -135,15 +135,25 @@ int ArkodeSolver::init(int nout, BoutReal tstep) {
   bool use_precon, use_jacobian, use_vector_abstol,set_linear;
   BoutReal start_timestep, max_timestep, min_timestep,fixed_timestep;
   bool imex,expl,impl; // Time-integration method
-  int MXSUB = mesh->xend - mesh->xstart + 1;
+
+  // Compute band_width_default from actually added fields, to allow for multiple Mesh objects
+  //
+  // Previous implementation was equivalent to:
+  //   int MXSUB = mesh->xend - mesh->xstart + 1;
+  //   int band_width_default = n3Dvars()*(MXSUB+2);
+  int band_width_default = 0;
+  for (auto fvar : f3d) {
+    band_width_default += fvar.var->getNx();
```

Comment:
Isn't `MXSUB+2 /= Field3D{}.getNx()` and if so is this change intentional (or rather why is it intended)?

Reply:
I'm a bit confused about what this setting is doing, but you're right, I should leave the current behaviour.